gcloud/logging/client.py

```diff
@@ -14,9 +14,11 @@
 
 """Client for interacting with the Google Cloud Logging API."""
 
-
 from gcloud.client import JSONClient
 from gcloud.logging.connection import Connection
+from gcloud.logging.connection import _LoggingAPI
+from gcloud.logging.connection import _MetricsAPI
+from gcloud.logging.connection import _SinksAPI
```

Comment:
The [logging v2 protos](https://github.com/googleapis/googleapis/blob/master/google/logging/v2/) define three services:

- `LoggingServiceV2` (defined in https://github.com/googleapis/googleapis/blob/master/google/logging/v2/logging.proto)
- The misnamed `ConfigServiceV2` (defined in https://github.com/googleapis/googleapis/blob/master/google/logging/v2/logging_config.proto)
- `MetricsServiceV2` (defined in https://github.com/googleapis/googleapis/blob/master/google/logging/v2/logging_metrics.proto).